sky/setup_files/setup.py

```diff
@@ -68,7 +68,7 @@ def parse_footnote(readme: str) -> str:
     'PrettyTable',
     # Lower local ray version is not fully supported, due to the
     # autoscaler issues (also tracked in #537).
-    'ray[default]>=1.9.0,<=1.13.0',
+    'ray[default]>=1.9.0',
     'rich',
```

Comment:
Should we pin <= 2.0? To avoid ray releasing a new version that accidentally breaks us (there are many beta interfaces).